packages/config-service/src/services/globalConfig.ts

```diff
@@ -495,10 +495,11 @@ export class GlobalConfig {
       required: false,
       defaultValue: null,
     },
-    NPM_PACKAGE_VERSION: {
+    // the actual env var in the node process is npm_package_version
+    npm_package_version: {
```

Comment:
Q: is this highlighting a bug in the Config Service?
Or is this by design and an oversight in the initial implementation?

Reply:
I would say it was a simple oversight in the initial implementation.  In a node context the environment variable is called `npm_package_version`. All lowercase.  By making this simple update no conditional logic, that was already tested and working, needed to change.